src/sentry/conf/server.py

```diff
@@ -2900,3 +2900,27 @@ def build_cdc_postgres_init_db_volume(settings):
 
 # Show session replay banner on login page
 SHOW_SESSION_REPLAY_BANNER = False
+
+# Mapping of (logical topic names, slice id) to physical topic names
+# and kafka broker names. The kafka broker names are used to construct
+# the broker config from KAFKA_CLUSTERS. This is used for slicing only.
+# Example:
+# SLICED_KAFKA_TOPICS = {
+#   ("generic_metrics", 0): {
```

Comment:
Earlier in this file we see the line `KAFKA_SNUBA_GENERIC_METRICS = "snuba-generic-metrics"`. What is "generic_metrics" and where does this "logical" topic name come from since no concept of a different logical topic name previously existed in Sentry?

Should this be:
```suggestion
#   (KAFKA_SNUBA_GENERIC_METRICS, 0): {
```

Reply:
I was keeping the example generic since it can be used for any topic name. I have changed it to `KAFKA_SNUBA_GENERIC_METRICS` now.